src/main/java/de/digitalcollections/iiif/hymir/image/business/ImageServiceImpl.java

```diff
@@ -35,13 +34,57 @@
 import javax.imageio.stream.ImageInputStream;
 import javax.imageio.stream.ImageOutputStream;
 import org.imgscalr.Scalr;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
 @Service
 public class ImageServiceImpl implements ImageService {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(ImageServiceImpl.class);
+
+  /**
+   * @param image buffered image to check for alpha channel
+   * @return true, if image contains alpha channel
+   * @see <a
+   *     href="https://docs.oracle.com/javase/8/docs/api/java/awt/image/ColorModel.html#hasAlpha--">Javadoc
+   *     ColorModel</a>
+   */
+  public static boolean containsAlphaChannel(BufferedImage image) {
+    return image.getColorModel().hasAlpha();
+  }
+
+  /**
+   * @param image buffered image to check for transparent pixels
+   * @return true, if image contains transparent pixels
+   */
+  public static boolean containsTransparency(BufferedImage image) {
```

Comment:
~~Das ist ziemlich ineffizient, gibt's da nicht was effizienterers in der ImageIO? Reicht es nicht zu checken ob das Bild einen Typ hat der Alpha-Kanäle supported?~~

Das ist doch eine Utility-Methode die nur in den Unit-Tests verwendet wird, was macht die im `ImageService`? Bzw. nachdem der Unit-Test ja nur diese Methode testet die im eigentlichen Code nirgendwo verwendet wird, kann das nicht komplett raus?

Reply:
ja, kann. anfangs wusste ich nicht, ob nur alpha-channel reicht. hat aber gereicht. also dann lösch ich es

Reply:
ok, ist draussen